src/friendly-to-api.ts

```diff
@@ -76,8 +78,29 @@ export const friendlySyntaxToApiSyntax = (config: string): Required<Pick<Authori
         default:
           assertNever(definition.type);
       }
+
+      relationsMetadataMap[relationName] = {
+        directly_related_user_types: []
+      };
+
+      allowedTypes?.forEach((allowedType: string) => {
+        metadataAvailable = true;
+          let objectAndRelation = allowedType.split("#");
```

Comment:
Just for readability
```suggestion
          let [userType, usersetRelation] = allowedType.split("#");
          let toAdd: any = {
            "type": userType,
          };
          if (usersetRelation) {
            toAdd.relation = usersetRelation;
          }
```